BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Loop_Scripts/Asym_XF_Loop.sh

```diff
@@ -19,9 +19,9 @@
 module load python/3.6-conda5.2
 
 ####### VARIABLES: LINES TO CHECK OVER WHEN STARTING A NEW RUN ###############################################################################################
-RunName='OneBatchTest'	## This is the name of the run. You need to make a unique name each time you run.
-TotalGens=100			## number of generations (after initial) to run through
-NPOP=12			## number of individuals per generation; please keep this value below 99
+RunName='Test_2023_18_11'	## This is the name of the run. You need to make a unique name each time you run.
+TotalGens=2			## number of generations (after initial) to run through
+NPOP=5			## number of individuals per generation; please keep this value below 99
 Seeds=1			## This is how many AraSim jobs will run for each individual## the number frequencies being iterated over in XF (Currectly only affects the output.xmacro loop)
 FREQ=60				## the number frequencies being iterated over in XF (Currectly only affects the output.xmacro loop)
 NNT=2000			## Number of Neutrinos Thrown in AraSim   
```

Comment:
Pull request looks good. NNT in the loop script is now the total number of neutrinos per antenna. I had it to be the neutrinos per seed a couple versions ago, but the newest pr changed it back.

Reply:
Gotcha